README.md

```diff
@@ -0,0 +1,29 @@
+# gp-v2-services
+
+This repository contains backend code for [Gnosis Protocol V2](TODO high level gpv2 link) written in Rust.
+
+## Order Book
+
+The `orderbook` crate provides the http api through which users (usually through a frontend web application) interact with the order book. Users can add signed orders to the order book and query the state of their orders.
```

Comment:
```suggestion
The `orderbook` crate provides the http api through which users (usually through a frontend web application) interact with the order book.
Users can add signed orders to the order book and query the state of their orders.
```

Reply:
Not sure if Github is showing your suggestion properly but I think this has no effect on the rendered mark down. If the rendered markdown is supposed to start a new paragraph then we need to include a full empty line in the raw text.
I think that is what you want so I'll add the empty line.